src/components/features/actionsMenu/VendorTableActionsMenu.tsx

```diff
@@ -53,17 +119,39 @@ export default function VendorTableActionsMenu({
         open={open}
         onClose={handleClose}>
         {options.map((option) => (
-          <MenuItem key={option.value} onClick={() => actionHandler(option.value)}>
-            {translate(option.title)}
+          <MenuItem key={option} onClick={() => actionHandler(option)}>
+            {option}
           </MenuItem>
         ))}
       </Menu>
 
+      {selectedOption === 'Редактирай' && (
```

Comment:
dont have options in Bulgarian. make them in English. Like what you have done with the "move entry" one

Reply:
Fixed.